examples/nodejs/eventbridge/README.md

```diff
@@ -0,0 +1,54 @@
+# Welcome to Momento <-> Eventbridge Integration Example Project
+
+The project demonstrates a write-through cache pattern for DynamoDB using DynamoDB Streams, AWS EventBridge and Momento.
```

Comment:
Let's add a little more color here about what people will see and do in the app.